docs/content/integrations/duckdb/reference.mdx

```diff
@@ -50,7 +112,7 @@ When Dagster materializes `sepal_data` and loads the `iris_dataset` asset using
 
 ---
```

Comment:
[1] No rule is needed here, since the following heading is an H3.

Reply:
ty ty i keep forgetting this